sunpy/map/sources/mlso.py

```diff
@@ -35,10 +35,11 @@ def __init__(self, data, header, **kwargs):
         # Fill in some missing info
         self.meta['observatory'] = 'MLSO'
         self.meta['detector'] = 'KCor'
-        self.meta['waveunit'] = 'nanometer'
+        self.meta['waveunit'] = self.meta.get('waveunit', 'nanometer')
```

Comment:
Should this not be "nm"?
```suggestion
        self.meta['waveunit'] = self.meta.get('waveunit', 'nm')
```